TrackerMC/fcl/prolog.fcl

```diff
@@ -8,7 +8,7 @@ makeSGS : {
 makeSD : {
     module_type : StrawDigisFromStrawGasSteps
     TimeOffsets   : [  @sequence::CommonMC.TimeMaps ]
-    StrawGasStepModule : makeSGS
+    StrawGasStepModule : ""
```

Comment:
I do not understand this change, can you please explain?

Reply:
Before we stored StepPointMCs in each background frame and they were all read in a single makeSGS module to create a single StrawGasStepCollection. Now we store StrawGasSteps in each background frame and so makeSD needs to do the combining. This functionality already exists in makeSD and we supply an empty input tag to do it.